lnbits/tasks.py

```diff
@@ -110,8 +110,7 @@ async def internal_invoice_listener():
     """
     while settings.lnbits_running:
         checking_id = await internal_invoice_queue.get()
-        logger.info("> got internal payment notification", checking_id)
```

Comment:
why remove the loggers?

Reply:
its more descriptive inside the dispatcher, because notification does not mean anything happened yet

Reply:
done with a bit of a change